mathics/core/parser/parser.py

```diff
@@ -1328,7 +1339,7 @@ def p_Span(self, token):
         return self.e_Span(Number1, token, 0)
 
     def p_String(self, token: Token) -> String:
-        result = String(unescape_string(token.text))
+        result = String(token.text[1:-1])
```

Comment:
Avoiding "unescape_string" probably addresses the warnings in testing we've been getting about invalid Python escape sequences.

I think the situation previously was that the scanner was relying on `unescape_string` to convert some escape strings rather than handed these in the scanner.